### ISSUE TITLE
Overwrites {{ build }} jinja

### ISSUE THREAD

**Issue by CJ-Wright** [regro/cf-scripts#32]
@tkelman commented on [Fri Feb 23 2018](https://github.com/regro/cf-graph/issues/26)

see https://github.com/conda-forge/awscli-feedstock/pull/29

---

@CJ-Wright commented on [Fri Feb 23 2018](https://github.com/regro/cf-graph/issues/26#issuecomment-368188923)

Thank you for reporting!

---

@CJ-Wright commented on [Fri Feb 23 2018](https://github.com/regro/cf-graph/issues/26#issuecomment-368190833)

This came about because you were able to update the version before the bot realized there is a new version. Normally that PR would have included a version bump.

@scopatz thoughts?
This is being overwritten because in the replacements a) there is no `{{ build ...}` in our replacement setup, b) there is also a replacement which overwrites the build setting. I don't suppose that there is a meaningful way to do this in the regex?

To be fair, having the build number as a jinja variable is a bit uncommon, especially as it is usually only used once, hence why we didn't include it in the initial setup.

---

@scopatz commented on [Mon Feb 26 2018](https://github.com/regro/cf-graph/issues/26#issuecomment-368615773)

Well, you could check and make sure that the `build: NUM` really is just a a simple number before overwriting it.  If it isn't you can punt.

---

@scopatz commented on [Mon Feb 26 2018](https://github.com/regro/cf-graph/issues/26#issuecomment-368616302)

Basically, this comes down to the fact that the ticker is assuming that there is a specific form of meta.yaml, and if it isn't in that form, it will fail
